include/aikido/robot/ConcreteRobot.hpp

```diff
@@ -224,6 +224,10 @@ class ConcreteRobot : public Robot
   void setCRRTPlannerParameters(
       const util::CRRTPlannerParameters& crrtParameters);
 
+  // Returns a pointer of the RNG belonging to this robot
+  // The ConcreteRobot is still responsible for cleaning this RNG, so use it only while the ConcreteRobot is alive!
+  aikido::common::RNG* getRNGPtr();
```

Comment:
We already have  `std::unique_ptr<aikido::common::RNG> cloneRNG()` defined below. Let's move that one to public and remove this, so that the user of the cloned RNG doesn't have to worry about the lifecycle of the robot.